pkg/morph/client/audit/client.go

```diff
@@ -25,7 +27,12 @@ const (
 	listByNodeResultsMethod  = "listByNode"
 )
 
-// New creates, initializes and returns the Client instance.
-func New(c *client.StaticClient) *Client {
-	return &Client{client: c}
+// NewFromMorph returns the wrapper instance from the raw morph client.
+func NewFromMorph(cli *client.Client, contract util.Uint160, fee fixedn.Fixed8, opts ...client.StaticClientOption) (*Client, error) {
+	sc, err := client.NewStatic(cli, contract, fee, opts...)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return nil, fmt.Errorf("could not create static client of audit contract: %w", err)
```

to sync with the other constructors

Reply:
Fixed.